app/controllers/medicaid/income_job_number_controller.rb

```diff
@@ -21,7 +21,11 @@ def number_of_jobs
     end
 
     def skip?
-      not_employed?
+      multi_member_household? || not_employed?
+    end
+
+    def multi_member_household?
```

Comment:
we have a similar method in another controller (thinking single_member_household). wonder if we want to just move these up a few levels, maybe just into the StepsController, since they seem generally useful?

Reply:
agree! see my other comment re: `ApplicationController` ?